src/main/scala/io/iohk/praos/domain/Block.scala

```diff
@@ -0,0 +1,21 @@
+package io.iohk.praos.domain
+
+import io.iohk.praos.crypto.{Hasher, Seed, Signature, VrfProof, combineSeeds}
+
+
+case class Block(
+  state       : Option[Hasher#Digest],
+  slotNumber  : SlotNumber,
+  data        : List[Transaction],
```

Comment:
Maybe we should rename this field to `transactions` or something similar?

Reply:
Maybe. I stuck to the Praos paper, which seems to assume that a block may contain any data (since the protocol doesn't care about block contents), even though ultimately it will contain transactions. Following the same reasoning, If we want to rename `data` to `transactions`, then we should also rename 'state' to 'prevBlockHash' or similar.

Reply:
Discussed with Nico, we will stick to the Praos paper.